### PR TITLE
Change of  the word Tier to Service Plan from all success and error messages in the API responses

### LINKED ISSUE #1280: Change the word `Tier` to `Service Plan` from all success and error messages in the API responses
**Issue/Feature Description:**
Many of the API responses for service plans have the word `Tier` in them.
service-plan-backends-exist-message-2
service-plan-backends-exist-message
service-plan-name-exists-message-2
service-plan-name-exists-message

**Why this issue to fixed / feature is needed(give scenarios or use cases):**
The word `tier` in the messages will be confusing to the user since the feature has been renamed to service plan.

All the occurrences of the word `tier` in the API responses have to be changed to `service plan`.

**How to reproduce, in case of a bug:**

**Other Notes /  Environment Information: (Please give the env information, log link or any useful information for this issue)**

### PR DESCRIPTION
**Pull request by vineela1999**
**What type of PR is this?**
 kind bug fix

**What this PR does / why we need it**:
This PR is to change the word tier to service plan in error messages and success messages in API responses 
**Which issue(s) this PR fixes**:

Fixes #1280 

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
kind TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

**Review by rajat-soda — APPROVED** [review]
LGTM

File: api/pkg/backend/service.go [hunk 1]
```diff
@@ -703,15 +703,15 @@ func (s *APIService) UpdateTier(request *restful.Request, response *restful.Resp
...
+		errMsg := fmt.Sprintf("some backends in AddBackends request: %v already exists in service plan", tenantExist)
 		log.Error(errMsg)
 		response.WriteError(http.StatusBadRequest, errors.New(errMsg))
 		return
 	}
 	log.Info("Check Tenant should exists before removing")
 	tenantNotExist := utils.ResourcesCheckBeforeRemove(res.Tier.Tenants, updateTier.DeleteTenants)
 	if len(tenantNotExist) > 0 {
-		errMsg := fmt.Sprintf("failed to update tier because delete backends: %v doesnt exist in tier", tenantNotExist)
+		errMsg := fmt.Sprintf("failed to update tier because delete backends: %v doesnt exist in service plan", tenantNotExist)
```
Comment (PravinRanjan10) [inline api/pkg/backend/service.go:714]: failed to update tier ==> failed to update service plan

File: api/pkg/backend/service.go [hunk 2]
```diff
@@ -694,7 +694,7 @@ func (s *APIService) UpdateTier(request *restful.Request, response *restful.Resp
 	log.Info("Check backends should exists before removing")
 	backendsNotExist := utils.ResourcesCheckBeforeRemove(res.Tier.Backends, updateTier.DeleteBackends)
 	if len(backendsNotExist) > 0 {
-		errMsg := fmt.Sprintf("failed to update tier because delete backends: %v doesnt exist in tier", backendsNotExist)
+		errMsg := fmt.Sprintf("failed to update tier because delete backends: %v doesnt exist in service plan", backendsNotExist)
```
Comment (PravinRanjan10) [inline api/pkg/backend/service.go:697]: failed to update tier ==>failed to update service plan

**Review by PravinRanjan10 — CHANGES_REQUESTED** [review]
Since, you are modiying CRUD part of SSP, please update log.Info() also. And update the tier with SSP in log.Info().

File: api/pkg/backend/service.go (hunk 1 above)
Comment (vineela1999) [reply api/pkg/backend/service.go]: changed

File: api/pkg/backend/service.go (hunk 2 above)
Comment (vineela1999) [reply api/pkg/backend/service.go]: changed

**Comment by vineela1999** [comment]
> Since, you are modiying CRUD part of SSP, please update log.Info() also. And update the tier with SSP in log.Info().

modified

**Review by PravinRanjan10 — APPROVED** [review]
LGTM

**Review by kumarashit — APPROVED** [review]
LGTM

**Review by anvithks — APPROVED** [review]
LGTM

**Pull request merged**